### DISCUSSION TITLE
Laravel 11

### DISCUSSION THREAD

**Discussion by gradecost2** [Laravel-Backpack/community-forum#975] [Q&A (Help)] [answered]
Hello.

Does Laravel-Backpack officially support laravel 11?

In documentation only laravel 10:
https://prnt.sc/iThVo2_hHiVa
https://prnt.sc/Dg2Wb-aFgh49
https://prnt.sc/i7XtbAb6FNRm

But we find this article https://github.com/Laravel-Backpack/community-forum/discussions/896

We have some big projects and our clients must understand all risks of updating.

Thanks!

**Comment by pxpm** [comment, marked as answer]
Hey there @gradecost2 thanks for the question and sorry for the confusion.

Backpack v6 supports Laravel 10 or 11. https://github.com/Laravel-Backpack/CRUD/blob/1fe921d1b1dbabafcad587e49582e13c453cf9ae/composer.json#L38

That's the "minimum required version" to run Backpack v6 on your system. We should probably make it more clear in our docs, thanks for the heads up. 🙏 

If you have clients on Laravel 10 + Backpack v6, upgrading to Laravel 11 should be just updating your `composer.json` and any dependencies needed.

You can have a look at our demo project `composer.json` that we already updated to Laravel 11 and uses pretty much all Backpack packages https://github.com/Laravel-Backpack/demo/blob/main/composer.json

If you are updating from Backpack v5, you should check the https://backpackforlaravel.com/docs/6.x/upgrade-guide and weight the changes needed in those projects. 

Start with one, see how it goes, we are here to help if needed. 🙏 

Cheers
